airmozilla/manage/static/manage/js/survey-edit.js

```diff
@@ -31,19 +31,28 @@ $(function() {
     // hijack the ability to save/edit
     $('form.edit').submit(function() {
         var form = $(this);
+        $(form.children('button')).text('Saving..');
         var tr = form.closest('tr');
         var textarea = $('textarea', tr);
+        $(textarea).attr('readonly','readonly');
         var data = {
             csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').val()
         };
         data[textarea.attr('name')] = textarea.val();
         $.post(form.attr('action'), data)
         .then(function(response) {
+
+            $(textarea).removeAttr('readonly');
             if (response.error) {
                 alert('Unable to save. Error: ' + response.error);
             } else {
                 textarea.val(response.question);
+                $(form.children('button')).text('Saved');
             }
+            setTimeout(function()
```

Comment:
This might sound silly and picky but the style format we use everywhere else is:
```javascript
setTimeou(function() {
    ...
}, X * 1000);
```
I.e. after the `function()` there's always one space followed by a `{`.